scripts/gallery/numpy/team.md

```diff
@@ -0,0 +1,141 @@
+## NumPy Team Gallery
+
```

Comment:
This file has a mix of tabs and spaces that would be good to fix (all spaces preferred).

More importantly, I think the raw html should be a partial; `layouts/partials/team.html` would be the place to put it. You only need one for a single gallery; the same partial can then be reused under `Maintainer`, `Web Team`, etc.

Reply:
ok - will look into it.

Reply:
Ralf, as discussed via email dated 03/30, I will work on the new issue to remove jinja2 dependency. https://github.com/numpy/numpy.org/issues/213  For now, the team.html is generated by numpy_team.py script in scripts/gallery and it stores generated html in Hugo's static/gallery area.  This page is linked to the about page from Hugo main page.  I have cleaned up content/en/team.md and placed in in script/gallery folder as it has jinja2 constructs.